encode.go

```diff
@@ -35,7 +35,11 @@ func encode(parameterStatus *parameterStatus, x interface{}, pgtypOid oid.Oid) [
 
 		return []byte(v)
 	case bool:
-		return []byte(fmt.Sprintf("%t", v))
+		if v {
+			return []byte{'t'}
+		}
+
+		return []byte{'f'}
```

Comment:
Elsewhere we use `strconv.AppendBool()`. We should probably do one or the other, not both.